src/App.js

```diff
@@ -1,23 +1,31 @@
-import logo from './logo.svg';
+import { useEffect, useState } from 'react';
 import './App.css';
+import axios from "axios"
+import {LandingPage,Homepage} from "./Pages/Pages"
```

Comment:
in the pages, folder name the pages.js as index.js that way you can default import it means, here instead of doing ./Pages/Pages you could have just written ./Pages that's it if only that file would be index,js instead of pages,js

Reply:
Sure I will  do from now on